src/Lean/PrettyPrinter/Delaborator/Builtins.lean

```diff
@@ -100,11 +100,17 @@ def delabConst : Delab := do
           c := `_root_ ++ c
         else
           c := c₀
-      return mkIdent c
+      mkIdent c
```

Comment:
This changes the behavior of the code (previously it was an early return).  But I think the new behavior is correct, since universe parameters don't have an effect on whether we need an explicit (`@`) annotation.